### PR TITLE
Fix missing changelogs

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* Leverage AddLegacySource api from OpenTelemetrySDK instead of
+  ActivitySourceAdapter
+  ([#1836](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1836))
+
```
Comment (reyang) [inline src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:8]: Could we remove the empty lines here?

File: src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* Leverage AddLegacySource api from OpenTelemetrySDK instead of
```
Comment (reyang) [inline src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:5]: What does this mean for the reader? (what do we expect the reader to get)?

Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:5]: For most users who just used the instrumentation without any customization - probably nothing.
But for advanced users who updates to this version (from prior) will see differences - eg: Sampling/kicking off processor pipeline is now done by the TracerProvider itself, before instrumentation gets the activity. So if one wrote a custom sampler or custom ActivityProcessor.OnStart, that would see activity earlier, even before its Kind/Source/DisplayName is updated.

Comment (reyang) [reply src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:5]: This seems to be a better description that we want to add to changelog? "Sampling/kicking off processor pipeline is now done by the TracerProvider itself, before instrumentation gets the activity."

Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:5]: I'll adjust this.

File: src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
...
+  ActivitySourceAdapter
+  ([#1836](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1836))
+* Performance optimization
+  by leveraging sampling decision and short circuiting path.
+  ([#1903](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1903))
+
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:11]: Test comment

File: src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Leverage AddLegacySource api from OpenTelemetrySDK instead of
+  ActivitySourceAdapter
+  ([#1836](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1836))
+* Performance optimization
+  by leveraging sampling decision and short circuiting path.
+  ([#1904](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1904))
+
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md:11]: Test comment

File: src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md (hunk 2 above)
Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md]: Reworded.
@utpilla we need to update the PR description in these PRs to reflect user impact - like the Samplers/Processor.OnStart being called *before* instrumentation.

**Review by reyang — APPROVED** [review]
LGTM.

**Pull request merged**